### PR TITLE
Lab 11 Bluetooth

### PR DESCRIPTION
**Pull request by chasegriswold**
Hey Alex/Miguel, we're hoping to get a code review from you for Lab 11. Alex, you were a huge help with this one, thank you so much :)

### CODE REVIEW

File: server/gatt_counter.c [hunk 1]
```diff
@@ -89,7 +90,7 @@ const uint8_t adv_data[] = {
     // Flags general discoverable
     0x02, BLUETOOTH_DATA_TYPE_FLAGS, APP_AD_FLAGS,
     // Name
-    11, BLUETOOTH_DATA_TYPE_COMPLETE_LOCAL_NAME, 'A', 's', 'h', 't', 'o', 'n', ' ', 'B', 'L', 'E',
+    8, BLUETOOTH_DATA_TYPE_COMPLETE_LOCAL_NAME, 'D', 'o', 'o', 'm', 'B', 'o', 't',
```
Comment (Mgomez-01) [inline server/gatt_counter.c:93]: Best name for the bluetooth haha. DOOMBOT! 🤖🤖🤖🤖🤖🤖🤖🤖🤖

File: server/picow_bt_example_freertos.c [hunk 2]
```diff
@@ -34,6 +34,10 @@ static void packet_handler(uint8_t packet_type, uint16_t channel, uint8_t *packe
             gap_local_bd_addr(local_addr);
             printf("BTstack up and running on %s.\n", bd_addr_to_str(local_addr));
             break;
+        case BTSTACK_EVENT_NR_CONNECTIONS_CHANGED:
+            uint8_t devices = btstack_event_nr_connections_changed_get_number_connections(packet);
```
Comment (Mgomez-01) [inline server/picow_bt_example_freertos.c:38]: I spent too long looking for this get number connections thing.

**Review by Mgomez-01 — APPROVED** [review]
Looks good to me!  Really cool to work with Bluetooth in this way, but it can be a pain!

**Review by Cromie1 — APPROVED** [review]
Considering I watched you two complete this I completely approve everything. Happy we are all done. Enjoy the break!

**Pull request merged**